pygmt/tests/test_grdfill.py

```diff
@@ -114,6 +114,31 @@ def test_grdfill_gridfill_dataarray(grid):
     npt.assert_array_equal(result[3:6, 3:5], bggrid[3:6, 3:5])
 
 
+def test_grdfill_hole(grid, expected_grid):
+    """
+    Test grdfill with different hole values.
+    """
+    # Prepare for a grid with -99999 as holes.
```

Comment:
Maybe this sounds nicer.

```suggestion
    # Prepare for a grid with a node value of -99999 for holes.
```